integration-tests/gatsby-cli/__tests__/build.js

```diff
@@ -12,8 +12,9 @@ describe(`gatsby build`, () => {
   it(`creates a built gatsby site`, () => {
     const [code, logs] = GatsbyCLI.from(cwd).invoke(`build`)
 
-    logs.should.contain(`success open and validate gatsby-configs`)
-    logs.should.contain(`success load plugins`)
+    logs.should.contain(
+      `success open and validate gatsby-configs, load plugins`
+    )
```

Comment:
I merged 2 activities together as loading config and plugins really seem like single step (and it was just cleaner code-wise after putting it in single function to be executed by worker) - can backtrack from it to go back to current 2 steps (or maybe just figure out better label than currently used one for merged activity)